public/locales/en/uploadDatasetFiles.json

```diff
@@ -1,3 +1,6 @@
 {
-  "breadcrumbActionItem": "Upload Files"
+  "breadcrumbActionItem": "Upload Files",
+  "cancel": "Cancel upload",
+  "info": "Drag and drop files here.",
+  "select": "Select Files to Add"
```

Comment:
Should these all be "files" (lowercase) or "Files" (Title Case)?

Right now it's a mix.

Reply:
I replaced the Title Case with lower case (except the first letter of the first word). I think it is better now.